index.ts

```diff
@@ -9,14 +9,32 @@ const bucketName: string = process.env.BUCKET_NAME as string;
 const getImage = async (event: EventDto): Promise<any> => {
 	try {
 		// parse params
+		const imageKey: string | undefined = event?.pathParameters?.imageName;
```

Comment:
should validate `event?.pathParameters != null` (“!= null" will check for both null and undefined)
so that we don’t have to use `.?` below the check. 
the control flow also clearer and safer.